README.md

```diff
@@ -0,0 +1,9 @@
+# Delegated Auth
+
+Delegate authentication and user management to another WordPress site. Support native WordPress logins via the login form, and OAuth2 REST API requests. The AOuth2 REST API plugin must be enabled on the delegated upstream site.
+
+When a login request is delegated to the upstream WordPress site a local cache of the user will be inserted into the WordPress database, mirroring all data, roles and capabilities. The login is synchronized on every request with the upstream WordPress site to make sure data is fresh and capabilities have not been removed. This can add some performance lag on all authenticated requests.
```

Comment:
btw, is this something we could move to Cavalcade eventually?

Reply:
Yeah I'm not sure - it would need to do it for all users, which TBH might not be a bad thing anyway. Unless you mean could it just defer the check out of thread, in which case yes could be good.